docs/bokeh/source/docs/user_guide/basic/data.rst

```diff
@@ -483,7 +481,7 @@ In the example below, the data set ``flowers`` contains a categorical variable
 called ``species``. All data belongs to one of the three species categories
 ``setosa``, ``versicolor``, or ``virginica``. The second plot in this example
 uses a |GroupFilter| to only display data points that are a member of the
-category ``setosa``:
+category ``versicolor``:
```

Comment:
Also entirely up to your interest, but just in case: we are trying to phase out the use of the iris dataset everywhere, in favor of this penguins dataset. See this example for usage in a different context:

https://docs.bokeh.org/en/latest/docs/examples/basic/data/transform_markers.html

Ideally the `filter_group.py` example is updated to use penguins instead of iris, and this text is updated to match.